packages/kbn-optimizer/limits.yml

```diff
@@ -46,7 +46,7 @@ pageLoadAssetSize:
   lens: 96624
   licenseManagement: 41817
   licensing: 29004
-  lists: 228500
+  lists: 280504
```

Comment:
NOTE: This is hopefully temp situation. As we add more things to packages and break down packages in size we should be able to reduce this.

The UI elements from this plugin might need to move back to security_solutions and then we do react.lazy there to load things or async imports.